File: engine/engine-model/src/main/scala/net/server/channel/packet/cash/operation/WorldTransferPacket.scala
package net.server.channel.packet.cash.operation

class WorldTransferPacket(private var _action: Int, private var _itemId: Int, private var _newWorldId: Int) extends BaseCashOperationPacket(_action = _action) {
  def itemId: Int = _itemId

  def newWorldId: Int = _newWorldId
}

File: engine/engine-model/src/main/java/net/server/channel/packet/cash/operation/WorldTransferPacket.java
package net.server.channel.packet.cash.operation;

public class WorldTransferPacket extends BaseCashOperationPacket {
   private final Integer itemId;

   private final Integer newWorldId;

   public WorldTransferPacket(Integer action, Integer itemId, Integer newWorldId) {
      super(action);
      this.itemId = itemId;
      this.newWorldId = newWorldId;
   }

   public Integer itemId() {
      return itemId;
   }

   public Integer newWorldId() {
      return newWorldId;
   }
}
